src/qt/bitcoingui.cpp

```diff
@@ -412,6 +419,25 @@ void BitcoinGUI::createActions()
                 action->setEnabled(false);
             }
         });
+        connect(m_restore_wallet_action, &QAction::triggered, [this] {
+            QString backup_file = GUIUtil::getOpenFileName(this,
+                                                          //: The title for Restore Wallet File Windows
+                                                          tr("Load Wallet Backup"), QString(),
+                                                          //: The file extension for Restore Wallet File Windows
+                                                          tr("Wallet Data File (*.dat)"), nullptr);
```

Comment:
This line and this one https://github.com/bitcoin-core/gui/blob/c99f58a07ecc85291a140b12c71fd2a9870a005e/src/qt/walletview.cpp#L216 must be consistent. Although, have no preference which one is better.

And please separate untranslatable file extension from the translatable string.

Reply:
Done in https://github.com/bitcoin-core/gui/pull/471/commits/e7a3f698b5f3f7dde8632c4911abd4e5bc1f06cb.
The file extension is now separated from the file format name.